src/app/models/AudioEvent.ts

```diff
@@ -47,6 +50,8 @@ export class AudioEvent
   @bawPersistAttr()
   public readonly audioRecordingId?: Id;
   @bawPersistAttr()
+  public readonly channel?: number;
```

Comment:
channel isn't writable - but it should be. File a bug and comment out

Reply:
Related issue: https://github.com/QutEcoacoustics/baw-server/issues/717